src/draive/conversation/completion/state.py

```
@@ -54,44 +53,52 @@ async def completion(
         *,
         instruction: Instruction | str | None = None,
         input: ConversationMessage | Multimodal,  # noqa: A002
```

Pylint (3.3.7)

[warning] Redefining built-in 'input'

(W0622)